taskcluster/android_taskgraph/parameters.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from taskgraph.parameters import extend_parameters_schema
+from voluptuous import All, Any, Range, Required
+
+
+def get_defaults(repo_root):
+    return {
+        "pull_request_number": None,
+        "release_type": "",
+    }
+
+
+extend_parameters_schema(
+    {
+        Required("pull_request_number"): Any(All(int, Range(min=1)), None),
+        Required("release_type"): str,
+    },
+    defaults_fn=get_defaults,
+)
```

Comment:
Nit: Let's remove the call from there:

https://github.com/mozilla-mobile/firefox-android/blob/191d5fd17d0a9d610770db7668ea6563d715effa/taskcluster/android_taskgraph/__init__.py#L19-L23

We may want to add `next_version` too here.

Reply:
Ouch, I didn't notice that even though I even touched that file!

Hmm, I wonder why running `taskgraph` locally didn't pick up those defaults? I agree that the way I have it implemented is the "official way" of doing it though.. So we can go with that.